py3status/modules/dropboxd_status.py

```diff
@@ -3,55 +3,87 @@
 Display status of Dropbox daemon.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    format: prefix text for the dropbox status (default 'Dropbox: {}')
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default "Dropbox: {status}")
+    status_busy: show when Dropbox is busy (default None)
+    status_off: show when Dropbox isn't running (default "isn't running")
+    status_on: show when Dropbox is up to date (default "Up to date")
 
-Valid status values include:
+Value for `status_on` if left None:
     - Dropbox isn't running!
+Value for `status_off` if left None:
+    - Up to date
+Values for `status_busy` if left None:
+    - Connecting...
     - Starting...
     - Downloading file list...
     - Syncing "filename"
-    - Up to date
+
+Format placeholders:
+    {status} Dropbox status
 
 Color options:
-    color_bad: Dropbox is unavailable
-    color_degraded: All other statuses
-    color_good: Dropbox up-to-date
+    color_bad: Not running
+    color_degraded: Busy
+    color_good: Up to date
 
 Requires:
-    dropbox-cli: command line tool
+    dropbox-cli: command line interface for dropbox
 
 @author Tjaart van der Walt (github:tjaartvdwalt)
 @license BSD
 """
-
```

Comment:
these lines should not be deleted. see #740 

You are getting the hang of stuff so I'm going to start expecting higher quality PRs